src/main/java/io/vertx/ext/hawkular/VertxHawkularOptions.java

```diff
@@ -410,4 +414,28 @@ public VertxHawkularOptions addDisabledMetricsType(MetricsType metricsType) {
   public boolean isMetricsTypeDisabled(MetricsType metricsType) {
     return this.disabledMetricsTypes.contains(metricsType);
   }
+
+  public VertxHawkularOptions setFeedId(String feedId) {
+    this.feedId = feedId;
+    return this;
+  }
+
+  public String getFeedId() {
+    if (feedId == null) {
+      throw new RuntimeException("feed id is null.");
```

Comment:
In the constructor? How? since people would code like this
```java
VertxHawkularOptions options = new  VertxHawkularOptions();
options.setFeedId("foo");
options.setRootResourceId("bar");
```
The code above is actually valid but it's gonna throw the exception since we test the validity in the constructor.